### DISCUSSION TITLE
add new source help

### DISCUSSION THREAD

**Discussion by Tileyukcj** [mampfes/hacs_waste_collection_schedule#2136] [General]
Hi,
https://preston.gov.uk/ - This is the council i live in that doesnt seem to be listed in the sources. Would this be an easy addition to add? 
Bin details are at this url-
view-source:https://selfservice.preston.gov.uk/service/Forms/ShowWasteCalendar.aspx?Id=20240609214100606d5a86f417406b&UPRN=100010542409
also attached a pic of the source of that page for reference.
Any help appreciated :)
![source](https://github.com/mampfes/hacs_waste_collection_schedule/assets/100723304/833793c2-6ba4-4d82-8dd3-a04feccd063c)

**Comment by 5ila5** [comment]
Having the UPRN as URL Parameter is generally a good sign for easy implementation, but your screenshot shows variables like VIEWSTATE and EVENTVALIDATION  which could make it a lot harder. I currently cannot load the page, does it work for you? (temporary outage or geo-blocked for Germany?)

**Comment by 5ila5** [comment]
There is already a request: #1939 
Which probably means I already looked at it, and it was too much work to do it right away. I will look at it again if I can access the webpage

**Comment by Tileyukcj** [comment]
The link does work but takes me back to an address selector. 
https://selfservice.preston.gov.uk/service/Forms/FindMyNearest.aspx?Service=bins
can you get to that page? If not I presume it’s a geo block.

**Reply by 5ila5** [reply]
No seems to be geo-blocked (does not load with Tor with GB exit node either). I currently do not have a VPN, so I cannot look into it at the moment
